src/Psalm/StatementsSource.php

```diff
@@ -39,7 +39,7 @@ public function getRequireNesting(): int;
 
     public function isStatic(): bool;
 
-    public function getSource(): ?StatementsSource;
+    public function getSource(): StatementsSource;
```

Comment:
None of the child return null here, plus it was creating an incoherence between interfaces.